src/renderer/shared/ui/Popovers/Popover/Popover.tsx

```diff
@@ -1,22 +1,26 @@
 import { Popover as Popup, Transition } from '@headlessui/react';
-import { AriaRole, Fragment, PropsWithChildren, ReactNode, useId, useRef, useState } from 'react';
+import { AriaRole, Fragment, PropsWithChildren, ReactNode, useEffect, useId, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
 
 import { cnTw } from '@shared/lib/utils';
 import { useDebounce } from '@shared/lib/hooks';
+import { popoverUtils } from '../common/utils';
 
 type Props = {
   content: ReactNode;
   offsetPx?: number;
   panelClass?: string;
   contentClass?: string;
+  horizontalPos?: 'left' | 'center' | 'right';
```

Comment:
we don't have vertical position, maybe call it `position` like in other components?